### PR TITLE
Ensure tablet button shows up; protections for controller mapping; Ensure instructions show

### PR DESCRIPTION
**Pull request by zfox23**
This will be tested during the main test plan of #10875, so there's no test plan here.

### CODE REVIEW

**Comment by zfox23** [comment]
[When I start the PR I don't have the spectator Cam App in my tablet (is it because I started in desktop mode?)](https://trello.com/c/hXtXEfpU/236-when-i-start-the-pr-i-dont-have-the-spectator-cam-app-in-my-tablet-is-it-because-i-started-in-desktop-mode)

**Comment by zfox23** [comment]
[In the spectator cam app, I get Desktop instructions (Press Control+O) rather than HMD instructions to switch previews](https://trello.com/c/RG5XdAGL/237-in-the-spectator-cam-app-i-get-desktop-instructions-press-controlo-rather-than-hmd-instructions-to-switch-previews)

File: scripts/system/spectatorCamera.js [hunk 1]
```diff
@@ -149,7 +149,7 @@
                 button.clicked.connect(onTabletButtonClicked);
             }
         } else if (button) {
-            if (!showSpectatorInDesktop || isShuttingDown) {
+            if ((!isHMDMode && !showSpectatorInDesktop) || isShuttingDown) {
```
Comment (davidkelly) [inline scripts/system/spectatorCamera.js]: 👍

File: scripts/system/spectatorCamera.js [hunk 2]
```diff
@@ -326,11 +329,16 @@
...
-            controllerType = "Vive";
-        } else if (VRDevices.includes("OculusTouch")) {
-            controllerType = "OculusTouch";
+        if (VRDevices) {
+            if (VRDevices.includes("Vive")) {
+                controllerType = "Vive";
+            } else if (VRDevices.includes("OculusTouch")) {
+                controllerType = "OculusTouch";
+            } else {
+                return; // Neither Vive nor Touch detected
```
Comment (davidkelly) [inline scripts/system/spectatorCamera.js]: we don't need to call setMonitorShowsCameraViewAndSendToQML before returning?  Just curious -- have not followed the logic all the way to the qml side for this...

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1517/HighFidelity-Beta-PR10935-4a74522884b8576f0243eb9e5bcb064be1db0dea.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1517/HighFidelity-Beta-PR10935-4a74522884b8576f0243eb9e5bcb064be1db0dea.dmg

File: scripts/system/spectatorCamera.js (hunk 2 above)
Comment (zfox23) [inline scripts/system/spectatorCamera.js]: Good questions:
Regarding `setMonitorShowsCameraViewAndSendToQML()`: That function isn't actually called in the body of this function; it's called in the controller mapping.
Regarding `updateControllerMappingCheckbox`: You made me think that there's a better way to do this, so I'll change the code.

**Pull request merged**